tc/sgx/trusted_worker_manager/enclave/kme/ext_work_order_info_kme.h

```diff
@@ -180,9 +184,21 @@ class ExtWorkOrderInfoKME : public ExtWorkOrderInfoImpl {
     }
 
 private:
+
+    /*
+     * Calculates hash on the encrypted keys, input-data-keys and output-data-keys
+     * @param wo_key_info - Instance of WorkOrderKeyInfo
+     * @param wo_key_info_hash [OUT] - Computed hash value
+    */
     void CalculateWorkOrderKeyInfoHash(WorkOrderKeyInfo wo_key_info,
         ByteArray& wo_key_info_hash);
 
+    /*
+     * Creates JSON document of encryption keys, signing keys
```

Comment:
Did you intend to say "serialized JSON object" by "JSON document"?

Reply:
Yes, will rephrase accordingly